### PR TITLE
Docs 4054 cloudhub tab in settings

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: modules/ROOT/pages/monitoring_settings_page.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+= Anypoint Monitoring Settings
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+On the Settings page, you can:
+
+* Find the instructions to install Anypoint Monitoring on an on-premises server to monitor applications running on that server.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/monitoring_settings_page.adoc]: Do they only monitor Mule applications or all apps on the server? (don't need to say if context is set already).

File: modules/ROOT/pages/quick-start.adoc [hunk 2]
```diff
@@ -46,14 +42,9 @@ To use Anypoint Monitoring, you need this permission:
 Typically, administrators set permissions through Access Management. See xref:access-management::users.adoc#granting-permissions-and-roles-to-users[Granting Permissions and Roles to Users].
 
 [[runtime_versions]]
-== Supported Platform and Mule Versions
+== Supported CloudHub Mule Versions
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/quick-start.adoc]: Should we say "Supported Versions of Mule Deployed to CloudHub"?

File: modules/ROOT/pages/quick-start.adoc [hunk 3]
```diff
@@ -46,14 +42,9 @@ To use Anypoint Monitoring, you need this permission:
...
 
-For Anypoint Monitoring to start monitoring your Mule apps, you must deploy them
-to the appropriate version of Mule runtime.  Note that the Anypoint Monitoring
-images (for example, 3.9.1-AM) receive the latest patch updates once per month,
-while the non-AM (mainline) images receive patch updates as soon as they are
-available. If your production environment requires a Mule runtime patch, MuleSoft
-recommends that you switch to a mainline runtime.
+For Anypoint Monitoring to start monitoring your Mule apps, you must deploy them to the appropriate version of Mule runtime engine.  Anypoint Monitoring images (for example, 3.9.1-AM) receive the latest patch updates once per month, while the non-AM (mainline) images receive patch updates as soon as they are available. If your production environment requires a Mule runtime engine patch, you should switch to a mainline runtime.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/quick-start.adoc]: not sure what mainline means here.

File: modules/ROOT/pages/enable_apps_deployed_to_cloud.adoc [hunk 4]
```diff
@@ -0,0 +1,15 @@
+= Enable Mule Applications Deployed to CloudHub
```
Comment (andrewb-ms) [inline modules/ROOT/pages/enable_apps_deployed_to_cloud.adoc]: We need to divide this up a bit I think.  There are three ways to enable.

1.  Via the UI (what you have)
2. By adding this property into the properties tab in the CloudHub settings page:

anypoint.platform.config.analytics.agent.enabled=true

3.  For CI/CD customers, they can do it by setting this property during their CI/CD deployment via the Anypoint CLI:

anypoint.platform.config.analytics.agent.enabled=true

**Pull request merged**